README.md

```diff
@@ -1,4 +1,4 @@
-# Bena dialect survey
+# CLDF dataset derived from Mitterhofer's "Dialect Survey of Bena" from 2013
```

Comment:
Bena Dialect Survey is the title. Not Dialect Survey of Bena.

Reply:
sorry, yes, you are right, the book gives this title and never uses that phrase, good suggestion.